Foundation/NSFileManager.swift

```diff
@@ -343,7 +343,19 @@ open class FileManager : NSObject {
         This method replaces fileSystemAttributesAtPath:.
      */
     open func attributesOfFileSystem(forPath path: String) throws -> [FileAttributeKey : Any] {
-        NSUnimplemented()
+        var s = statfs()
+        guard statfs(path, &s) == 0 else {
```

Comment:
the path here does have a slight problem; non-ascii characters of file names probably will cause issues. In the objc version we use effectively the same idea as `fileSystemRepresentationWithPath(:String)`

However this shortcoming shouldn't stop this PR imho